api/apps/__init__.py

```
@@ -68,6 +69,23 @@ def _unauthorized_message(error):
+    user_lang = getattr(user, "language", None) if user is not None else None
+    loc = resolve_locale(request.headers.get("Accept-Language"), user_lang, os.environ.get("LANG"))
+    set_locale(loc)
+    g.locale = loc
+
+
+@app.before_request
+async def bind_locale():
+    user = None
+    try:
+        user = _load_user()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)